src/base_station_base.h

```diff
@@ -22,13 +22,13 @@ extern StationPool _station_pool;
 struct StationSpecList {
 	const StationSpec *spec;
 	uint32 grfid;      ///< GRF ID of this custom station
-	uint8  localidx;   ///< Station ID within GRF of station
+	uint16_t localidx; ///< Station ID within GRF of station
```

Comment:
is using the _t types something we want to start doing? imo being inconsistent is worse than updating everything all at once (if we even end up doing that)

Reply:
:shrug: They are being used elsewhere, although actually in some cases that was 10 years ago...